PHPGraphQLArticle.md

```diff
@@ -152,43 +146,39 @@ query UserPosts{
 }
 ```
 
-That will return an array of posts with only those two fields: id and title. So we got to annother improvement over rest here: 
+This will return an array of posts with only those two fields: id and title. So we got to annother improvement over REST here: *In GraphQL the client can specify in the query what fields are needed so only those are returned in the response.*
 
-*In GraphQL the client can specify in the query what fields are needed and only those will be returned on the response.*
-
-Now, let's move into another direction and think that we need more info. We are looking at a list of articles and we want to see those articles and the last 5 comments on them. In rest, we could do something like: 
+Let's consider another scenario. We are looking at a list of articles and want to the last 5 comments for each of them. In REST, we could do something like: 
```

Comment:
This is not another scenario, but we are deepening on the scenario above. Getting more info for each post. What about: "Let's move further"... or something like that?